src/symposium-tickets/symposium-tickets-button.html

```diff
@@ -0,0 +1,32 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="./symposium-tickets-firebase.html">
+<link rel="import" href="./symposium-tickets-payments.html">
+
+
+<dom-module id="symposium-tickets">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+
+    </style>
+
+    <section>
+      <symposium-tickets-payments></symposium-tickets-payments>
+      <symposium-tickets-firebase></symposium-tickets-firebase>
+
+      <button disabled>Buy!</button>
```

Comment:
Yes. Since scaffold-index can only use plain HTML and we "upgrade" later with a custom element